frontend-react/src/components/FileHandlers/FileHandlerMessaging.tsx

```diff
@@ -60,7 +61,19 @@ export const RequestedChangesDisplay = ({
         data.some((responseItem) => responseItem.message);
 
     function handleSaveToCsvClick() {
-        return saveToCsv(data, {
+        const dataWithErrorMessage = data.map((item) => {
```

Comment:
Talked with Stephen, instead of creating a separate string component or using `dangerouslySetInnerHTML`, we decided to go for the simplest, most direct route and import `renderToString`.